sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1412,6 +1412,8 @@ def delete_blobs(  # pylint: disable=delete-operation-wrong-return-type
         """
         if len(blobs) == 0:
             return iter([])
+        if self.is_azurite is True:
+            kwargs['azurite_name'] = self.account_name
```

Comment:
This is how I am passing along the name, looking for feedback if this is a good idea